packages/config/src/scripts/dev-script.js

```diff
@@ -40,9 +40,8 @@ async function devScript(argv, cwd) {
     } else {
       configPath = resolve(__dirname, './dev.webpack.config.js');
     }
-    processArgs.push(`node_modules/.bin/webpack serve -c ${configPath}`);
     await setEnv(cwd);
-    spawn('node', processArgs, {
+    await spawn(`npm exec -- webpack serve -c ${configPath}`, processArgs, {
```

Comment:
We can't use the await, there are additional commands that can run in parallel bellow.

Reply:
👍